src/mantarray_file_manager/files.py

```diff
@@ -474,24 +470,24 @@ def get_raw_reference_reading(self) -> NDArray[(2, Any), int]:
         return self._raw_ref_reading
 
     def _check_for_trimmed_file(
-        self, times: NDArray[(1, Any), int], time_delta_centimilliseconds: int
+        self, times: NDArray[int], time_delta_centimilliseconds: int
```

Comment:
is there a reason you dropped the details from the `NDArray` type definition?

Reply:
oh that was a mistake that I forgot to change back!